amqp/src/main/scala/akka/stream/alpakka/amqp/model.scala

```diff
@@ -121,15 +121,25 @@ object AmqpConnectionUri {
 }
 
 final case class AmqpConnectionDetails(
-    host: String,
-    port: Int,
+    hostAndPortList: List[(String, Int)],
     credentials: Option[AmqpCredentials] = None,
     virtualHost: Option[String] = None,
-    sslProtocol: Option[String] = None
+    sslProtocol: Option[String] = None,
+    requestedHeartbeat: Option[Int] = None,
+    connectionTimeout: Option[Int] = None,
+    handshakeTimeout: Option[Int] = None,
+    shutdownTimeout: Option[Int] = None,
+    networkRecoveryInterval: Option[Int] = None,
+    automaticRecoveryEnabled: Option[Boolean] = None,
+    topologyRecoveryEnabled: Option[Boolean] = None,
+    exceptionHandler: Option[ExceptionHandler] = None
```

Comment:
I also added a `addHostAndPort(hostAndPorts: (String, Int))` and the Java one `addHostAndPort(hostAndPort: akka.japi.Pair[String, Int])` to add host/port combinations to the already given list. I am a bit unsure about the name `addHostAndPort`. What do you think?